synapse/lib/layer.py

```diff
@@ -245,6 +247,17 @@ async def getIden(self):
 
 EDIT_PROGRESS = 100   # (used by syncIndexEvents) (<etyp>, (), ())
 
+INDX_NORM = b'\x00\x00'
```

Comment:
should we just jump directly to making these abrvs?

Reply:
Yeah I'm including these in the abrvs now

Reply:
like this:
```
def getIndxAbrv(self, indx, form=None, prop=None, tag=None):
        return self.indxabrv.bytsToAbrv(s_msgpack.en((indx, form, prop, tag)))
```